src/NLog/Internal/ReflectionHelpers.cs

```diff
@@ -349,7 +349,7 @@ public static bool IsValidPublicProperty(this PropertyInfo p)
 
         public static object GetPropertyValue(this PropertyInfo p, object instance)
         {
-#if NET45
+#if NET4_5
```

Comment:
oops looks like a bug (before)

Reply:
Yes was scanning for places where NET4_5 was used directly. I guess we should always opt-out, instead of having these opt-in situations that bites.

Reply:
Yes opt out is preferred, also when removing platforms